pkgs/build-support/build-fhs-chrootenv/env.nix

```diff
@@ -129,7 +129,7 @@ let
   setupLibDirs_multi = ''
     mkdir -m0755 lib32
     mkdir -m0755 lib64
-    ln -s lib32 lib
+    ln -s lib64 lib
```

Comment:
Wait... I guess multi-build can only mean that the main arch is 64-bit? If so, then I take back my question :-)